projects/storefrontapp-e2e-cypress/cypress/helpers/quote.ts

```diff
@@ -267,8 +585,41 @@ export function checkLinkedItemInViewport(index: number) {
  * Register GET quote route.
  */
 export function registerGetQuoteRoute(shopName: string) {
-  cy.intercept({
-    method: 'GET',
-    path: `${Cypress.env('OCC_PREFIX')}/${shopName}/users/current/quotes/*`,
-  }).as(GET_QUOTE_ALIAS.substring(1)); // strip the '@'
+  trace(
+    registerGetQuoteRoute.name,
+    () => {
+      cy.intercept({
+        method: 'GET',
+        path: `${Cypress.env('OCC_PREFIX')}/${shopName}/users/current/quotes/*`,
+      }).as(GET_QUOTE_ALIAS.substring(1)); // strip the '@'
+    },
+    arguments
+  );
+}
+
+/**
+ * Creates a simple log with ##### comment <functionName> ######
+ * @param comment Could be the description of the function
+ * @param functionName Name of the called function
+ */
+function log(comment: string, functionName: string) {
+  cy.log(`##### ${comment} <${functionName}> #####`);
+}
+
+/**
+ * Creates a Cypress log before function gets executed printing out name
+ * and args and another log after finishing the function.
+ * @param functionName Name of the function that should be logged
+ * @param func The function that is called
+ * @param args Arguments the function will be called with
+ */
+export function trace(functionName: string, func: Function, args?: Object) {
```

Comment:
This method won't be used anymore. Please, remove it

Reply:
done